src/main/java/mate/academy/internetshop/controllers/CompleteOrderController.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.internetshop.controllers;
```

Comment:
```suggestion
package mate.academy.internetshop.controller;
```
пекеджі називаються в однині